thumbor/loaders/file_loader.py

```diff
@@ -31,8 +41,10 @@ def load(context, path, callback):
 
             media.buffer = f.read()
             media.metadata.update({
-                'LastModified': datetime.utcfromtimestamp(stats.st_mtime)
+                'LastModified': datetime.utcfromtimestamp(stats.st_mtime),
+                'ContentLength': stats.st_size
```

Comment:
This value is actually not needed because the `buffer` itself holds the true value of the length.